src/main/java/com/example/paradise/domain/post/application/PostService.java

```diff
@@ -0,0 +1,92 @@
+package com.example.paradise.domain.post.application;
+
+import com.example.paradise.domain.post.domain.Post;
+import com.example.paradise.domain.post.domain.PostRepository;
+import com.example.paradise.domain.post.dto.CreatePostRequestDto;
+import com.example.paradise.domain.post.dto.PostResponseDto;
+import com.example.paradise.domain.post.dto.UpdatePostRequestDto;
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class PostService {
+
+    private final PostRepository postRepository;
+
+    static private Long userId = 1L;
+    static private String username = "홍길동";
+
+    // 1. 게시글 등록
+    public PostResponseDto createPost(CreatePostRequestDto requestDto) {
+        // dto -> entity
+        Post post = new Post(userId,username, requestDto.getContent());
+        
+        // 저장
+        Post savedpost = postRepository.save(post);
+        
+        // dto 변환
+        PostResponseDto postResponseDto = new PostResponseDto(savedpost);
+        return postResponseDto;
+    }
+
+
+    // 2. 게시글 조회 - 전체
+    public Page<PostResponseDto> getAllPosts(Pageable pageable) {
+        // 게시글 페이징 처리
+        Page<Post> postPage = postRepository.findAll(pageable);
+        // entity -> responsedto
+        return postPage.map(PostResponseDto::new);
+    }
+
+    // 3. 게시글 조회 - 단건
+    public PostResponseDto getPostById(Long postId) {
+        // 유효한 게시글인지  확인
+        Post post = checkById(postId);
+
+        return new PostResponseDto(
+                post.getPostId(),
+                post.getContent(),
+                post.getUsername(),
+                post.getCreatedAt(),
+                post.getUpdatedAt()
+        );
+    }
+
+    //4. 게시글 수정
+    public void updatePost(Long postId, UpdatePostRequestDto requestDto) {
+        // 유효한 게시글인지  확인
+        Post post = checkById(postId);
+
+        // 게시글 내용 업데이트
+        post.updateContent(requestDto.getContent());
+
+        // 저장
+        Post updatedPost = postRepository.save(post);
```

Comment:
이 `postRepository.save(post);`를 `Post updatedPost`라는 변수에 저장해도 해당 변수를 쓸 일이 없는데 굳이 변수에 해당 값을 저장해 줄 필요는 없을 거 같아요! 
```suggestion
        postRepository.save(post);
```
이렇게만 해줘도 괜찮을 거 같습니다!

Reply:
네 삭제하겠습니다~